all-spark-kernels/Dockerfile

```diff
@@ -6,7 +6,7 @@ FROM jupyter/all-spark-notebook:97a5071c5775
 MAINTAINER Jupyter Project <jupyter@googlegroups.com>
 
 # Install Kernel Gateway
-RUN pip install 'jupyter_kernel_gateway>=1.0,<2.0'
+RUN pip install --no-cache-dir 'jupyter_kernel_gateway>=1.0,<2.0'
```

Comment:
Let's drop the restrictions on JKG version.

Reply:
Done as requested